molecule/config.py

```diff
@@ -393,7 +393,7 @@ def _get_defaults(self):
             },
             'scenario': {
                 'name':
-                'default',
+                os.path.split(os.path.dirname(self.molecule_file))[-1],
```

Comment:
Can we use some `os.path` function to avoid any possibility of `IndexError` reaching the user?

Reply:
This should work; I'm not sure if there's an even more direct way to express it.
```python
os.path.basename(os.path.dirname(self.molecule_file))
```

```python
>>> import os
>>> x = '/path/to/some/dir/molecule.yml'
>>> os.path.basename(os.path.dirname(x))
'dir'
```

It can also be `or`-guarded to default to `default`, e.g.
```python
>>> x = '/molecule.yml'  # but probably this should never happen, right?
>>> default_scenario_name = os.path.basename(os.path.dirname(x)) or 'default'
>>> default_scenario_name
'default'
```